language/diem-framework/modules/CRSN.move

```diff
@@ -24,8 +26,28 @@ module DiemFramework::CRSN {
         min_nonce: u64,
         size: u64,
         slots: BitVector,
+        force_shift_events: EventHandle<ForceShiftEvent>,
     }
 
+    /// Whenever a force shift is performed a `ForceShiftEvent` is emitted.
+    /// This is used to prove the absence of a transaction at a specific sequence nonce.
+    struct ForceShiftEvent has drop, store {
```

Comment:
Just to double check `store` here is to work with the phantom type parameter in the event handle?

Reply:
Correct! it needs a `T: drop + store` so this needs `store` to match up with those restrictions.